doc/introduction/configuration.rst

```diff
@@ -52,4 +52,39 @@ Configuration options
 
 **port (int)** (*optional*)
     The port to be used when connecting to the remote service.
-    Corresponding environment variable: ``SF_API_PORT``
\ No newline at end of file
+    Corresponding environment variable: ``SF_API_PORT``
+
+Store your account
+------------------
+
+Using the :func:`~.store_account` function, a configuration file can be created easily. It only requires specifying the authentication token. Apart from that, further configuration options can be passed as keyword arguments.
+
+Default configuration
+*********************
+
+The following is an example for using ``store_account`` with defaults:
+
+.. code-block:: python
+
+    import strawberryfields as sf
+    sf.store_account("MyToken")
+
+where ``"MyToken"`` contains the user-specific authentication token.
+
+It is advised to execute this code snippet **only once**, separately any other Python scripts.
```

Comment:
I'm not quite sure what you mean by this statement?

Reply:
I was trying to describe that a user should in most cases execute this code only once (and afterwards their configuration would be done) and that this code should be kept separate from other Strawberry Fields related code that they have (such that it does not get committed or shared). Just trying to help out with suggestions such that they wouldn't give away their tokens. Let me know if you would suggest rewording or including different ideas here!

Reply:
How about tying this in with a big warning not to commit your token?

Reply:
If the main message we want to convey is to treat your token carefully and not commit it, then that should be what is stated here :).